examples/key_release/Dockerfile.primary

```diff
@@ -1,15 +1,10 @@
 FROM mcr.microsoft.com/azureml/minimal-ubuntu20.04-py38-cpu-inference:20231102.v2
 
 USER root
-RUN apt update && apt upgrade -y && apt install -y wget curl jq python3-pip
+RUN apt update && apt upgrade -y && apt install -y python3-pip
```

Comment:
package manager cache can be cleared to make the image smaller
`apt-get clean && rm -rf /var/lib/apt/lists/*`